galileo_sdk/galileo_cli/missions.py

```diff
@@ -248,4 +266,91 @@ def create(name, public):
         spinner.stop()
         print("Created Mission:", mission.name)
         print("Mission ID: ", mission.mission_id)
-        print("Public: ", str(bool(public)))
\ No newline at end of file
+        print("Public: ", str(bool(public)))
+        
+        
+    @missions.command()
+    @click.option(
+        "-i",
+        "--id",
+        type=str,
+        multiple=False,
+        help="UUID of the Mission to update.",
+    )
+    @click.option(
+        "-n",
+        "--name",
+        type=str,
+        multiple=False,
+        help="Update the name of a Mission.",
+    )
+    @click.option(
+        "--ssid",
+        type=str,
+        multiple=False,
+        help="Cargo Bay ID for the input data of the Mission.",
+    )
+    @click.option(
+        "--dsid",
+        type=str,
+        multiple=False,
+        help="Cargo Bay ID for the output data of the Mission.",
+    )
+    @click.option(
+        "--spath",
+        type=str,
+        multiple=False,
+        help="Path within the source Cargo Bay to pull the input data of the Mission.",
+    )
+    @click.option(
+        "--dpath",
+        type=str,
+        multiple=False,
+        help="Path within the destination Cargo Bay to put the output data of the Mission.",
+    )
+    @click.option(
+        "--public/--private",
+        default=False,
+        help="Set the Mission to be either publicly or privately viewable.",
+    )
+    def update(id, name, ssid, dsid, spath, dpath, public):
+        """
+        Update the settings of a Mission in your account. 
+        """
+        
+        if not id:
+            print("Please specify a Mission with the -i or --id flag.")
+            return
+        
+        spinner = Halo("Finding Mission.", spinner="dot").start()
+        try:
+            mission = galileo.missions.list_missions(ids=[id])
+        except Exception as e:
+            spinner.stop()
+            print("Error:", e)
+            return
+        
+        spinner.stop()
+        
+        if len(mission) != 1:
```

Comment:
This might get weird if list_missions ever breaks such that more than one result is returned

Reply:
agreed, will revisit this later